gazelle/python/generate.go

```diff
@@ -239,16 +236,34 @@ func (py *Python) GenerateRules(args language.GenerateArgs) language.GenerateRes
 			}
 		}
 
-		pyLibrary = newTargetBuilder(pyLibraryKind, pyLibraryTargetName, pythonProjectRoot, args.Rel, pyFileNames).
+		pyLibrary := newTargetBuilder(pyLibraryKind, pyLibraryTargetName, pythonProjectRoot, args.Rel, pyFileNames).
 			addVisibility(visibility).
-			addSrcs(pyLibraryFilenames).
+			addSrcs(srcs).
 			addModuleDependencies(deps).
 			generateImportsAttribute().
 			build()
 
 		result.Gen = append(result.Gen, pyLibrary)
 		result.Imports = append(result.Imports, pyLibrary.PrivateAttr(config.GazelleImportsKey))
 	}
+	if cfg.PerFileGeneration() {
+		pyLibraryFilenames.Each(func(index int, filename interface{}) {
+			if filename == pyLibraryEntrypointFilename {
```

Comment:
What should happen if we have `bar/__init__.py` and `bar.py` in the same directory? Should gazelle throw an error? What does the Python spec says about such a case?

Reply:
I tested this just now and it only imports `bar/__init__.py`, ignoring `bar.py`

https://docs.python.org/3/tutorial/modules.html#packages says
> The import statement first tests whether the item is defined in the package; if not, it assumes it is a module and attempts to load it.